src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -13,6 +13,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_PROJECT = new Prefix("pr/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_EMPLOYEE = new Prefix("em/");
+    public static final Prefix PREFIX_PRIORITY = new Prefix("priority/");
```

Comment:
I think the prefix is too long, we'll need to shorten it so that users can use our applications more effectively. This can be done as a future enhancement though, it is not mission critical at the moment.